ale/rotation.py

```diff
@@ -0,0 +1,180 @@
+from scipy.interpolate import interp1d
+from scipy.spatial.transform import Slerp
+from scipy.spatial.transform import Rotation
+
+import numpy as np
+
+class ConstantRotation:
+    """
+    A constant rotation between two 3D reference frames.
+
+    Attributes
+    __________
+    source : int
+             The NAIF ID code for the source frame
+    dest : int
+           The NAIF ID code for the destination frame
+    """
+
+    def __init__(self, quat, source, dest):
+        """
+        Construct a constant rotation
+
+        Parameters
+        ----------
+        quat : array
+               The quaternion representation of the rotation as a numpy array
```

Comment:
The order of quaternions is a common point of confusion. It would be helpful to reference the order we expect the elements to be in. Maybe just a simple (x,y,z,w) or is it the other way?

Reply:
I'm just using scipy's order which is constant-last (x, y, z, w). I'll add this to the docs.